core/services/job/job_orm_test.go

```diff
@@ -603,7 +603,7 @@ func TestORM_CreateJob_OCRBootstrap(t *testing.T) {
 	legacyChains := evmrelay.NewLegacyChainsFromRelayerExtenders(relayExtenders)
 	jobORM := NewTestORM(t, db, legacyChains, pipelineORM, bridgesORM, keyStore, config.Database())
 
-	jb, err := ocrbootstrap.ValidatedBootstrapSpecToml(testspecs.OCRBootstrapSpec)
+	jb, err := ocrbootstrap.ValidatedBootstrapSpecToml(fmt.Sprintf(testspecs.OCRBootstrapSpec, uuid.New()))
```

Comment:
given that this pattern is used over and over, would it be better to a func the wraps it and hides the uuid call?

as a reader, i can't know what the uuid means since we are doing a sprintf and as a writer, i have to go to the spec to know how many args to give the sprintf, so might as well hide these details